packages/teams-js/test/public/chat.spec.ts

```diff
@@ -184,8 +203,50 @@ describe('chat', () => {
 
           const openChatMessage = utils.findMessageByFunc('chat.openChat');
           expect(openChatMessage).not.toBeNull();
-          expect(openChatMessage.args).toEqual([chatResponse]);
+          expect(openChatMessage.args).toEqual([normalizedChatRequest]);
+        });
+
+        it(`should successfully pass chatRequest to non-legacy Teams host when only one UPN is specified - Context: ${context}`, async () => {
```

Comment:
>it [](http://example.com/codeflow?start=8&length=2)

This is a new test to validate the existing behavior where `openGroupChat` delegates to `openChat` for a single user. This test validate that the request has the right shape (a single entry in an array)